new/t2.js

```diff
@@ -0,0 +1,25 @@
+/*eslint no-debugger: "error"*/
+
+function isTruthy1(x) {
+    debugger;
+    return Boolean(x);
+}
+
+function isTruthy2(x) {
+    debugger;
+    return Boolean(x);
+}
+
+var x = 5;;
```

Comment:
Unnecessary semicolon.

```suggestion
var x = 5;
```